---
layout: archive
title: "Software"
permalink: /software/
author_profile: true
---


The softwares developed by   my lab  are listed as follows: 

* [**TransferTWAS**](https://github.com/daoyuan-lai/TransferTWAS):  TransferTWAS: A transfer learning framework for cross-tissue transcriptome-wide association study
 [[paper]](https://doi.org/10.1016/j.ajhg.2025.06.006)



* [**MAAT**](https://github.com/wanghanmath/MAAT):  MAAT: a new nonparametric Bayesian framework for incorporating multiple functional annotations in transcriptome-wide association studies
 [[paper]](https://doi.org/10.1186/s13059-025-03485-x)


* [**TWAS.GKF**](https://github.com/AnqiWang2021/TWAS.GKF):  TWAS-GKF: a novel method for causal gene identification in transcriptome-wide association studies with knockoff inference [[paper]](https://doi.org/10.1093/bioinformatics/btae502)


* [**g-ldsc**](https://github.com/xzw20046/gldsc): Improved estimation of functional enrichment in SNP heritability using feasible generalized least squares [[paper]](https://pubmed.ncbi.nlm.nih.gov/38327050/)



* [**h2D2**](https://github.com/xiangli428/h2D2):  A Bayesian fine-mapping model using a continuous global-local shrinkage prior  h2D2 [[paper]](https://doi.org/10.1016/j.ajhg.2023.12.007)


* [**MRCI**]( https://github.com/zpliu/MRCI): Reciprocal causation mixture model for robust Mendelian randomization analysis using genome-scale summary data
[[paper]](https://doi.org/10.1038/s41467-023-36490-4)


* [**GraceAKO**](https://github.com/mxxptian/GraceAKO):  Grace-AKO: a novel and stable knockoff filter for variable selection incorporating gene network structures
   [[paper]](https://doi.org/10.1186/s12859-022-05016-y)


* [**GR2D2**](https://github.com/RavenGan/GR2D2):  The GR2D2 estimator for the precision matrices   [[paper]](https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbac426/6731716)


* [**TLMulti**](https://github.com/mxxptian/TLMulti): Multiethnic polygenic risk prediction in diverse populations through transfer learning
[[paper]](https://www.frontiersin.org/articles/10.3389/fgene.2022.906965/full)


* [**R2D2**](https://github.com/yandorazhang/R2D2): Bayesian Regression Using a Prior on the Model Fit: The R2-D2 Shrinkage Prior
  [[paper]](https://doi.org/10.1080/01621459.2020.1825449)
  


* [**CancerEffectSize**](https://github.com/yandorazhang/CancerEffectSize): Assessment of polygenic architecture and risk prediction based on common variants for cancers.  [[paper]](https://www.nature.com/articles/s41467-020-16483-3)



* [**GENESIS**](https://github.com/yandorazhang/GENESIS): GENetic Effect-Size distribution Inference from Summary-level data. [[paper]](https://www.nature.com/articles/s41588-018-0193-x) 




